apps/mkt/components/automation-carousel/automation-benefit-icon.tsx

```
@@ -0,0 +1,74 @@
+  const IconComponent = icons[type];
+
+  return (
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger asChild>
+          <div className="rounded p-[4px] bg-dark-600 border border-dark-400 flex items-center justify-center w-[28px] h-[28px]">
+            {React.cloneElement(IconComponent, { color, stroke, ...props })}
+          </div>
+        </TooltipTrigger>
+        <TooltipContent>
+          {type === "cycleTime" && <>Improves cycle time</>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)